### PR TITLE
chore: round timeToRefresh for logging.

### PR DESCRIPTION
**Pull request by raresraf**
Log the time duration for re-scheduling the refresh of ephemeral certificate rounded to milliseconds.

Right now, logging the current left duration for refresh is quite verbose. For instance, the message I see logged on my console why I try to connect to one instance is: `Scheduling refresh of ephemeral certificate in 54m59.294191713s`. I feel that this contains a little bit too much details. Rounding the timeToRefresh value during logging to ensure per-millisecond granularity should be good enough.

### CODE REVIEW

**Comment by enocom** [comment]
@raresraf Thanks for the PR. You'll need the sign the CLA in order for me to accept the PR.

**Comment by raresraf** [comment]
Signed.

File: proxy/proxy/client.go [hunk 1]
```diff
@@ -317,7 +317,7 @@ func (c *Client) startRefresh(instance string, refreshCfgBuffer time.Duration) c
 			timeToRefresh = certExpiration.Sub(now) - (5 * time.Second)
 			logging.Errorf("new ephemeral certificate expires sooner than expected (adjusting refresh time to compensate): current time: %v, certificate expires: %v", now, certExpiration)
 		}
-		logging.Infof("Scheduling refresh of ephemeral certificate in %s", timeToRefresh)
+		logging.Infof("Scheduling refresh of ephemeral certificate in %s", timeToRefresh.Round(time.Millisecond))
```
Comment (enocom) [inline proxy/proxy/client.go:320]: Let's just round to time.Second. The millisecond portion doesn't help much.

https://go.dev/play/p/g_fNIWzOi8J

Comment (raresraf) [reply proxy/proxy/client.go]: Sure. Done.

**Review by enocom — APPROVED** [review]

**Comment by enocom** [comment]
Thanks again!

**Pull request merged**